templates/security/OAuthAccessTokenManager.swift.hbs

```diff
@@ -99,12 +99,12 @@ actor OAuthAccessTokenManager {
                 resultData.createdAt = requestDate
                 self.accessToken = resultData
                 self.accessTokenDidChange?(resultData)
-            case 400...499:
+            case 400...428, 430...500:
```

Comment:
It's kind of curious that we allow the 429 retry to be configured but we hard-code this. I wonder whether that suggests that we shouldn't promote customising that retry status code... what use does someone have to change that except to do something that they shouldn't be doing.

How does our auth handle 429? Perhaps we should just have a 429 case above? Would that simplify this case?

I think this `case` needs a comment:

```swift
/// These status codes indicate a problem with the client's request, or an internal server error, which together we believe are reasonable to treat as meaning that the current refresh token is no longer viable.
```

Reply:
we do have a 429 case, it's the `default` case in this instance assuming that you still get a 429 after all those attempts. If you removed 429 from the retry configuration then it's still appropriate to respond with an unexpected response rather than invalidate the token.

Reply:
This is now a configuration option